documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -2516,25 +2516,47 @@ host    replication     <youruser>  ::1/128                 trust   // <3>
+==== PostgreSQL 15 or earlier clusters
 
-When using PostgreSQL 16 or newer, you can set up logical replication slots also on replica servers,
-and {prodname} does not need to connect to your primary server for ingesting change events.
-In comparison to connecting to the primary,
-connecting to a replica will yield slightly increased latencies, though.
+When you deploy {prodname} in environments that run PostgreSQL 15 or earlier, you can configure logical replication slots only on the primary server in the cluster.
+You cannot configure logical replication on replica servers in the cluster.
+Consequently, the {prodname} PostgreSQL connector can connect and communicate only with the primary server.
+If the primary server fails, the connector stops.
 
-When using PostgreSQL 17 or newer, you can set up logical replication slots on primary serves which are enabled for failover.
-The state of such a failover slot can automatically be propagated to one or more replica servers,
-allowing {prodname} to continue to ingest changes from a replica promoted to primary after a failure,
-without missing any events.
+Following a failure, after the cluster is repaired, you can once again promote the original primary server to `primary`, and then restart the connector.
+Alternately, you might promote a different PostgreSQL server to `primary`.
+Before you can promote a different server to primary, you must xref:setting-up-postgresql[properly configure] the server.
+Then, to enable the connector to consume from the new `primary` server, update properties in the connector configuration to match the details of the new server, for example, xref:postgresql-property-database-hostname[`database.hostname`], xref:postgresql-property-database-port[`database.port`], xref:postgresql-property-database-dbname[`database.dbname`], and xref:postgresql-property-slot-name[`slot.name`].
+After you update the configuration to point to the new primary, you can restart the connector.
 
-endif::community[]
+==== PostgreSQL 16 or later clusters
+
+When you deploy {prodname} with PostgreSQL 16 or later, you can set up logical replication slots on replica servers.
+In a PostgreSQL 16 cluster, {prodname} can capture change events from servers other than the primary server.
+However, be aware that {prodname} connections to replica servers generally experience higher latency than connections to a primary server.
```

Comment:
Perhaps we could pull up the tidbit from **Cluster failures** that emphasizes on PG16, you need to keep the replication slots on the replicas synchronized to support flawless failover post reconfiguration with Debeizum.

Reply:
Yes, again, I agree. Thanks. 
 ... Slowly putting the mind back into gear ...